src/plugin/__tests__/withAndroidConfiguration.test.ts

```diff
@@ -60,4 +60,15 @@ describe("withAndroidConfiguration", () => {
       expect(result.modResults.contents).toMatchSnapshot();
     });
   });
+  describe("with datadog gradle plugin version option", () => {
+    it("sets the provided version", async () => {
```

Comment:
I would add a case for setting the default version when no custom version is provided. I believe there used to be a test for this in `withAndroidProguardMappingFiles.test.ts`

Reply:
Thanks, good catch!